artiq/applets/plot_hist.py

```diff
@@ -10,6 +11,10 @@ class HistogramPlot(pyqtgraph.PlotWidget):
     def __init__(self, args):
         pyqtgraph.PlotWidget.__init__(self)
         self.args = args
+        self.flag = True
```

Comment:
``flag`` -> ``timer_triggered`` (with the opposite True/False values)?
Though:
1. This could be replaced with ``timer.isActive()``
2. Is it necessary to have this check at all?